pkg/apb/ext_creds_test.go

```diff
@@ -7,197 +7,23 @@ import (
-fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Region not specified. Unable to determine re
-gion from EC2_REGION."}
-        to retry, use: --limit @/opt/ansibleapp/actions/provision.retry
-PLAY RECAP *********************************************************************
-localhost                  : ok=2    changed=0    unreachable=0    failed=1
-`)
-	result, err := decodeOutput(output)
-	if err == nil {
-		t.Fatal("decode should've returned an error")
-	}
-
-	if result != nil {
-		t.Fatal("result should've been nil")
-	}
-	assertError(t, err, "provision failed, INSERT MESSAGE HERE")
-	// need to better parse the FAILED json returned.
-}
-
-func TestBuildExtractedCredentialsError(t *testing.T) {
-	output := []byte(`
-Login failed (401 Unauthorized)
-
-PLAY [all] *********************************************************************
-
-TASK [setup] *******************************************************************
-ok: [localhost]
-
-TASK [Bind] ********************************************************************
-changed: [localhost]
-
-TASK [debug] *******************************************************************
-ok: [localhost] => {
-    "msg": "<BIND_CREDENTIALS>eyJkYiI6ICJmdXNvcl9ndWVzdGJvb2tfZGIiLCAidXNlciI6ICJkdWRlcl90d28iLCAicGFzcyI6ICJkb2c4dHdvIn0="
-}
-
-PLAY RECAP *********************************************************************
-localhost                  : ok=3    changed=1    unreachable=0    failed=0
-`)
-	bd, _ := buildExtractedCredentials(output)
-	ft.AssertNotNil(t, bd, "credential is nil")
-}
-
 func TestBuildExtractedCredentials(t *testing.T) {
-	output := []byte(`
-Login failed (401 Unauthorized)
-
-PLAY [all] *********************************************************************
-
-TASK [setup] *******************************************************************
-ok: [localhost]
-
-TASK [Bind] ********************************************************************
-changed: [localhost]
+	output := []byte("eyJkYiI6ICJmdXNvcl9ndWVzdGJvb2tfZGIiLCAidXNlciI6ICJkdWRlcl90d28iLCAicGFzcyI6ICJkb2c4dHdvIn0=")
 
-TASK [debug] *******************************************************************
-ok: [localhost] => {
-    "msg": "<BIND_CREDENTIALS>eyJkYiI6ICJmdXNvcl9ndWVzdGJvb2tfZGIiLCAidXNlciI6ICJkdWRlcl90d28iLCAicGFzcyI6ICJkb2c4dHdvIn0=</BIND_CREDENTIALS>"
-}
-
-PLAY RECAP *********************************************************************
-localhost                  : ok=3    changed=1    unreachable=0    failed=0
-`)
 	bd, _ := buildExtractedCredentials(output)
 	ft.AssertNotNil(t, bd, "credential is nil")
 	ft.AssertEqual(t, bd.Credentials["db"], "fusor_guestbook_db", "db is not fusor_guestbook_db")
 	ft.AssertEqual(t, bd.Credentials["user"], "duder_two", "user is not duder_two")
 	ft.AssertEqual(t, bd.Credentials["pass"], "dog8two", "password is not dog8two")
 }
 
-func TestErrorDecodeOutput(t *testing.T) {
-	output := []byte(`
-	error: dial tcp [::1]:8443: getsockopt: connection refused
```

Comment:
Another useful test to ensure we properly return errors during a malformed input.